src/main/resources/schemas/single_justice_procedure.json

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
should we name this file public if its for the SJP public list? unless its a root SJP for the press list?

Reply:
Sounds good! I've updated the naming